main.py

```diff
@@ -13,32 +13,25 @@ def main():
     B.read()
     B.print()
 
+    C=A.mul(B)
+    C.print()
     
 
-    if A.c == B.r:
-        C = Matrix()
-        for i in range(A.r):
-            C.matrix.append(list([0] * B.c))
-
-        for i in range(A.r):
-            for j in range(B.c):
-                for k in range(A.c):
-                    C.matrix[i][j] += A.matrix[i][k] * B.matrix[k][j]
-
-        """for i in range(r1):
-            for j in range(c2):
-                C[i][j] += A[i][j]*B[j][i]"""
-
-        print("-------------------------------------------------")
-        print(f"The resultant matrix is:")
-        for i in range(A.r):
-            for j in range(B.c):
-                print(f"{C.matrix[i][j]}\t", end="")
-            print()
-
-    else:
-        print("The given matrices cannot be multiplied!")
-        print(f"Matrix A has {A.c} columns and Matrix B has {B.r} rows")
+
```

Comment:
This code fragment is not useful, can you remove it?

Reply:
Solved!